teachertool/src/components/CriteriaResultEntry.tsx

```diff
@@ -23,25 +23,33 @@ interface CriteriaResultNotesProps {
 
 const CriteriaResultNotes: React.FC<CriteriaResultNotesProps> = ({ criteriaId }) => {
     const { state: teacherTool } = useContext(AppStateContext);
+    const notesInitialValue = teacherTool.evalResults[criteriaId]?.notes;
 
     const onTextChange = (str: string) => {
         setEvalResultNotes(criteriaId, str);
     };
 
     return (
-        <div className={css["notes-container"]}>
-            <DebouncedTextarea
-                placeholder={lf("Write your notes here")}
-                ariaLabel={lf("Feedback regarding the criteria result")}
-                label={lf("Feedback")}
-                title={lf("Write your notes here")}
-                initialValue={teacherTool.evalResults[criteriaId]?.notes}
-                autoResize={true}
-                onChange={onTextChange}
-                autoComplete={false}
-                intervalMs={500}
-            />
-        </div>
+        <>
+            <div className={classList(css["notes-container"], "no-print")}>
+                <DebouncedTextarea
+                    placeholder={lf("Write your notes here")}
+                    ariaLabel={lf("Feedback regarding the criteria result")}
+                    label={lf("Feedback")}
+                    title={lf("Write your notes here")}
+                    initialValue={notesInitialValue}
+                    autoResize={true}
+                    onChange={onTextChange}
+                    autoComplete={false}
+                    intervalMs={500}
+                />
+            </div>
+            {notesInitialValue && (
```

Comment:
In this case, `notesInitialValue` is checking that notes exist on this criteria, correct? If that's true, I think we can just have the `notesInitialValue` variable just be `notes`.

Reply:
To clarify, I don't think having `InitialValue` in this variable name adds clarity, so I think having `notes` is good, an alternative could be `currentNotes`.

Reply:
I'm not quite sure what you're saying. Is this in regard to the name `notesInitialValue` itself? I chose this name to mesh cleanly with its original use: passing it DebouncedTextarea's `initialValue` prop. I factored the value out of that assignment for use in multiple places.

Reply:
Yes, sorry, just the name. I should've put my comment where the variable was first assigned. 

I like the fact that the name meshes with its original use, but I think because the notes are also being used in a spot (here) where we actually don't care if it's an "initialValue" or not, I think it should be a more general name to reflect that, thus the suggestion to rename the variable name to `currentNotes`.

Reply:
`currentNotes` seems less clear to me, as the value will change as the user edits the field. The name `initialNotesValue` makes sense in the screen rendering context where the value is editable, but not so much in the print context, where it can't change. I guess I can split the difference and just call it `notes` as you originally suggested.